1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh

```diff
@@ -213,6 +213,9 @@ pgbouncer_initialize() {
         if ! is_empty_value "$PGBOUNCER_AUTH_USER"; then
             database_value+=" auth_user=$PGBOUNCER_AUTH_USER"
         fi
+        if ! is_empty_value "$PGBOUNCER_CONNECT_QUERY"; then
+            database_value+=" connect_query='$PGBOUNCER_CONNECT_QUERY'"
```

Comment:
Use curly braces when concatenating env. vars with a regular string

```suggestion
            database_value+=" connect_query='${PGBOUNCER_CONNECT_QUERY}'"
```